[2021-10-16 13:04:58,472] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:56:00+00:00 [queued]>
[2021-10-16 13:04:58,541] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:56:00+00:00 [queued]>
[2021-10-16 13:04:58,542] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:04:58,542] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:04:58,542] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:04:58,601] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 10:56:00+00:00
[2021-10-16 13:04:58,625] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T10:56:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpv0k3gv_k', '--error-file', '/tmp/tmp4ug4uxgx']
[2021-10-16 13:04:58,626] {standard_task_runner.py:77} INFO - Job 267: Subtask get_gross_year_1
[2021-10-16 13:04:58,612] {standard_task_runner.py:52} INFO - Started process 484 to run task
[2021-10-16 13:04:58,741] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T10:56:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:04:58,950] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T10:56:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T10:56:00+00:00
[2021-10-16 13:04:58,986] {python.py:152} INFO - Done. Returned value was: ['$19.62 ', '2010\n']
[2021-10-16 13:04:59,084] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T105600, start_date=20211016T130458, end_date=20211016T130459
[2021-10-16 13:04:59,175] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:04:59,266] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
